index.html.in

```diff
@@ -411,28 +411,28 @@
                     </div>
                     <div class="modal-body">
                         <div data-text="What_Is_Apertium"></div>
-                        <div data-text="Maintainer" style="padding-bottom: 2em;"></div>
+                        <div data-text="Maintainer" id="maintainer"></div>
 
                         <div class="row lead">
                             <div class="col-md-6">
                                 <div class="center-block">
-                                    <a href="https://developers.google.com/open-source/soc/" target="_blank" style="text-decoration: none">
-                                        <img alt="Google Summer of Code" data-src="https://summerofcode.withgoogle.com/static/img/summer-of-code-logo.svg" style="height: 2.5em;">
+                                    <a href="https://developers.google.com/open-source/soc/" target="_blank" class="nodecoration">
+                                        <img alt="Google Summer of Code" data-src="https://summerofcode.withgoogle.com/static/img/summer-of-code-logo.svg" id="gsocImg">
                                     </a>
-                                    <a href="https://developers.google.com/open-source/gci/" target="_blank" style="text-decoration: none">
-                                        <img alt="Google Code-In" data-src="https://developers.google.com/open-source/gci/images/logo-icon.png" style="height: 2.5em; padding-left: .5em;">
+                                    <a href="https://developers.google.com/open-source/gci/" target="_blank" class="nodecoration">
```

Comment:
`no-decoration` in the Utilities section I mentioned below? Sorry, I read this file backwards.